smarthome/common/context_test.go

```diff
@@ -23,10 +23,9 @@ func TestContextProperty(t *testing.T) {
 	timeOfSample, _ := time.Parse(time.RFC3339, "2018-02-25T19:56:05+00:00")
 
```

Comment:
You can use time.Date() instead of time.Parse(). That is less readable, but avoids swallowing an error and be surprised by a zero time value. Or `t.Fatal()` on the parse error.